api/Navigator.json

```diff
@@ -657,7 +664,7 @@
                 "notes": "Not supported on macOS, see <a href='https://crbug.com/1144920'>bug 1144920</a>."
               },
               "firefox": {
-                "version_added": false
+                "version_added": "96"
```

Comment:
FYI @saschanaz I also added that text sharing is supported for share and canshare on desktop in the same versions as they were introduced. I have not added that it is supported for android but we can certainly do so once you merge https://github.com/mozilla-mobile/fenix/pull/22633